src/main.rs

```diff
@@ -17,18 +17,45 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+use clap::{CommandFactory, ErrorKind, Parser};
+
 use astarte_device_sdk::options::AstarteOptions;
 use astarte_device_sdk::registration;
 use astarte_device_sdk::AstarteDeviceSdk;
 
 use astarte_message_hub::config::MessageHubOptions;
 use astarte_message_hub::error::AstarteMessageHubError;
 
+/// A central service that runs on (Linux) devices for collecting and delivering messages from N
+/// apps using 1 MQTT connection to Astarte.
+#[derive(Parser, Debug)]
+#[clap(author, version, about, long_about = None)]
+struct Args {
+    /// Path to a valid .toml file containing the message hub configuration.
+    /// Do not provide this option with the `store_directory` option.
+    #[clap(short, long, value_parser)]
+    toml: Option<String>,
+    /// Directory used by the HTTP and ProtoBuf servers to retain the provided configuration.
```

Comment:
Leave this description more general, like:
```suggestion
    /// Directory used by Astarte-Message-Hub to retain configuration and other persistent things.
```

Reply:
But leaving the description more generic might be more confusing for the user.
The `store_directory` is used to retain only the configuration provided through HTTP or ProtoBuf. Not configuration passed with the `toml` option.

Reply:
Sure, but in future we can use this folder to store other information, such as a database for the AstarteMessageHub, etc.